### DISCUSSION TITLE
Python Kümeler (Sets) | SonsuzUs Github

### DISCUSSION THREAD

**Discussion by giscus** [sonsuzus/sonsuzus.github.io#301] [Site]
# Python Kümeler (Sets) | SonsuzUs Github

Kümeler

https://sonsuzus.github.io/posts/python_sets/

**Comment by akgembi12** [comment]
pop metodunu rastgele bir eleman silmesinin nedeni kumelerin sırasında olması ve sonunda bir eleman olmaması, değil mi
